src/ui/hooks/useToasts.ts

```diff
@@ -17,17 +23,28 @@ export default function useToasts(): UseToasts {
   const [toasts, setToasts] = useState<Toast[]>([]);
 
   const addToast = useCallback(
-    (message: string, variant: "success" | "danger") => {
+    (message: string, variant: "success" | "danger", permanent?: boolean) => {
       const id = window.crypto.randomUUID();
-      setToasts((oldToasts) => [...oldToasts, { id, message, variant }]);
-      setTimeout(() => {
-        setToasts((oldToasts) => oldToasts.filter((toast) => toast.id !== id));
-      }, 3500);
+      setToasts((oldToasts) => [
+        ...oldToasts,
+        { id, message, variant, permanent },
+      ]);
+      if (!permanent) {
+        setTimeout(() => {
+          setToasts((oldToasts) =>
+            oldToasts.filter((toast) => toast.id !== id),
+          );
+        }, 3500);
+      }
```

Comment:
We are doing this in `onClose` arg to `<Toast>`, so we dont need to do it here too.